client/webserver/site/src/js/markets.ts

```diff
@@ -968,16 +1043,21 @@ export default class MarketsPage extends BasePage {
       mkt.maxSellRequested = false
       mkt.maxSell = res.maxSell
       mkt.sellBalance = baseWallet.balance.available
-      this.setMaxOrder(res.maxSell.swap)
+      // Since this executes in callback quite a bit of time could have passed, and the
+      // user might have switched to another form already (from buy to sell order form,
+      // or vise-versa), update max value only if user is on the relevant form.
+      if (this.isSell()) {
+        this.setMaxOrder(res.maxSell.swap)
+      }
```

Comment:
I actually hit this while testing. This wouldn't be an issue were it not for the fact that single `Max` view gets re-used for both `buy` / `sell`. I was able to split many of Limit and Market related stuff into separate view, but splitting `buy` and `sell` just because of this issue doesn't seem worth it.